CurrencyDotNet.RestClient/Models/Requests/ClosePositionRequest.cs

```diff
@@ -1,26 +1,17 @@
-﻿using CurrencyDotNet.Common.Services;
-using CurrencyDotNet.RestClient.Interfaces;
+﻿using CurrencyDotNet.RestClient.Interfaces;
+using CurrencyDotNet.RestClient.Models.Requests.Abstractions;
 
 namespace CurrencyDotNet.RestClient.Models.Requests
 {
     /// <summary>
     /// Close an active leverage trade.
     /// </summary>
-    public class ClosePositionRequest : IRequestModel
+    internal class ClosePositionRequest : RequestModel, IRequestModel
```

Comment:
It would be better to inherit RequestModel from IRequestModel and then inherit other request models only form RequestModel.

Reply:
Yes, the base class implementation was added later, so the interface is redundant here, fixed that